book/website/foreward/history.md

```diff
@@ -0,0 +1,57 @@
+
+*The Turing Way* has been funded by The Alan Turing Institute since the very beginning.
+Due to close relationships with the Jupyter Project, the book was written in JupyterBook.
+A GitHub repository was made for the project, and internal processes decided in how to acknowledge authorship works within the project (i.e. installation of Welcome bot, All-Contributors bot) and collaborative spaces and events to contribute to the guides were created (such as the Collaboration Cafe and Book Dash).
+In-person workshops were given on tools aligned with the project (particulary Binder and JupterBook).
+The project was presented in many conferences across the UK and internationally.
+
+## Development and Growth (2019-2021)
+
+With the addition of Dr. Malvika Sharan in late 2019 as the community manager, *The Turing Way* continued to evolve and grow as it received more support and visibility.
+Malvika helped to incubate what are now key partnerships for ***The Turing Way*** within the wider 'open science' ecosystem, and helped to usher in the expansion of the project.
+This period corresponded with the COVID-19 pandemic lockdowns, which transitioned many processes of contribution, collaboration, and documentation into remote ones.
+For many, it also became clear that reproducibility on its own is not sufficient to address the major challenges (including incentive challenges!) faced by research data scientists.
+
+In part due to all these factors, *The Turing Way* was able to expand greatly, from one guide on reproducibility into five different guides related to project design, communication, collaboration, and research ethics.
+
+A global (and multilingual) community began to engage with *The Turing Way*, and the process of translation/localisation began informally.
+A Community Handbook was also established to document best practices that could be used by other communities.
+The Fireside Chat series was established in late 2021, facilitating shared spaces with networks and allies from across the open science ecosystem.
+The project received a few awards & citations in the 2021 London Mayor report, HiddenREF Awards, and Goldacre Report.
+The Book Dash Planning Committe was established in early 2022 to plan the community's flagship collaboration event.
+
+## Maintenance & Sustainability (2022-Ongoing)
+
+In 2022, Malvika and Kirstie became co-leads of *The Turing Way*, and welcomed Anne Lee Steele as the new community manager.
+
+As the number of contributors to the project has grown to over 400 people, *The Turing Way* has also developed into a kind of ["community of communities"](https://ben.balter.com/2019/07/18/a-community-of-communities-oscon-2019/).
+Many ongoing projects have been developing alongside the expansion of the guides, demonstrating the need for internal systems that enable the sustainability and maintenance of the project more broadly.
```

Comment:
I think it would be helpful to name some explicit examples here - from this sentence it's not clear what is meant by "many ongoing projects have been developing alongside the expansion of the guides" 

Do you mean that the project is providing the community infrastructure to support smaller scale, or offshooting collaborations? 

It's also not clear why this in itself demonstrates the need for internal systems around maintenance, I think those are two separate things so should be in two different sentences.